src/types.ts

```diff
@@ -267,7 +367,7 @@ export interface ILookerConnection {
    * Looker version.
    */
 
-  preload(): Promise<void>
+  preload(pushHistory?: boolean, options?: IConnectOptions): Promise<void>
```

Comment:
Would just using a partial version of LoadParams here make sense? Or at least support the single object rather than regular params